src/EventStore.LogV3/Raw.cs

```diff
@@ -87,5 +132,40 @@ public struct StreamTypeHeader {
 				set => _partitionId = value;
 			}
 		}
+
+		[StructLayout(LayoutKind.Explicit, Size = Size, Pack = 1)]
+		public struct StreamWriteHeader {
+			[FieldOffset(0)] public long _streamNumber;
+			[FieldOffset(8)] public long _startingEventNumber;
+			[FieldOffset(16)] public short _flags;
+			[FieldOffset(18)] public short _count;
+			[FieldOffset(20)] public int _metadataSize;
```

Comment:
is the naming correct?  (based on the log v3 specs, it's called system metadata size)